.gitignore

```diff
@@ -19,9 +19,6 @@
 # Ignore Intellij files
 .idea
 
-# Ignore Rails environment vars
-/config/initializers/environment_variables.rb
```

Comment:
@edemkumodzi I've reverted the changes changes I made earlier, maintaining only the update to env vars only.